streams/src/main/java/org/apache/kafka/streams/processor/internals/PartitionGroup.java

```diff
@@ -41,8 +41,8 @@
  *
  * PartitionGroup also maintains a stream-time for the group as a whole.
  * This is defined as the highest timestamp of any record yet polled from the PartitionGroup.
- * The PartitionGroup's stream-time is also the stream-time of its task and is used as the
- * stream-time for any computations that require it.
+ * Note however that any computation that depends on stream time tracks it on a per-operator basis to obtain an
+ * accurate view of the local stream time as seen by that node.
```

Comment:
`node` -> `processor`? Maybe we could call this "processor time" ?

Reply:
How about just `local stream time` -> ` local time` to avoid introducing too many types of time? Or do you feel "processor time" is clear enough (possibly more so than "local time" -- WDYT?)

Reply:
In light of renaming "stream time" -> "task time" I think it does make sense to call this "processor time" and establish a clear naming hierarchy